Int overflow in time_test.go on 32 bits arches

With Go 1.15.6, when running the tests on i686 and armv7hl:

```
github.com/aws/smithy-go/time
# github.com/aws/smithy-go/time [github.com/aws/smithy-go/time.test]
./time_test.go:9:50: constant 50520000000 overflows int
./time_test.go:45:49: constant 21123399936 overflows int
FAIL	github.com/aws/smithy-go/time [build failed]
```

--- a/time/time_test.go
+++ b/time/time_test.go
@@ -6,7 +6,7 @@ import (
 )
 
 func TestDateTime(t *testing.T) {
-	refTime := time.Date(1985, 4, 12, 23, 20, 0, int(50.52*float64(time.Second)), time.UTC)
+	refTime := time.Date(1985, 4, 12, 23, 20, 50, int(520*time.Millisecond), time.UTC)
 
 	dateTime := FormatDateTime(refTime)
 	if e, a := "1985-04-12T23:20:50.52Z", dateTime; e != a {
@@ -42,7 +42,7 @@ func TestHTTPDate(t *testing.T) {
 }
 
 func TestEpochSeconds(t *testing.T) {
-	refTime := time.Date(2018, 1, 9, 20, 51, 0, int(21.123399936*float64(time.Second)), time.UTC)
+	refTime := time.Date(2018, 1, 9, 20, 51, 21, 123399936, time.UTC)
 
 	epochSeconds := FormatEpochSeconds(refTime)
 	if e, a := 1515531081.1234, epochSeconds; e != a {

